adapter/internal/oasparser/envoyconf/http_filters.go

```diff
@@ -55,9 +53,28 @@ func getHTTPFilters() []*hcmv3.HttpFilter {
 	httpFilters := []*hcmv3.HttpFilter{
 		cors,
 		extAauth,
-		lua,
-		router,
 	}
+
+	logConf := config.ReadLogConfigs()
+
+	// debug log first filter
+	if logConf.DebugLogs.Enable {
```

Comment:
What if we define as 
if logConf.DebugLogs.Enable {
 httpFilters = []*hcmv3.HttpFilter{ cors, extAauth, getDebugLogFirstFilter(), lua, getDebugLogSecondFilter(), router}
} else {
 httpFilters = []*hcmv3.HttpFilter{ cors, extAauth, lua, router}
}

Reply:
IMO, it is more readable in this way.